src/app/pricing/vector/page.tsx

```diff
@@ -1,15 +1,27 @@
 "use client";
 
+import { generateFaqSchema } from "@/utils/structured-schema-generators";
+
 import PageHeaderDesc from "@/components/page-header-desc";
 import PageHeaderTitle from "@/components/page-header-title";
 import ProductToggle from "@/components/pricing/product-toggle";
 import CompareTable from "@/components/pricing/vector/compare-table";
 import Enterprise from "@/components/pricing/vector/enterprise";
 import PricingTable from "@/components/pricing/vector/pricing-table";
 
+import VectorFaqJson from "../../../../public/faq/vector.json";
```

Comment:
yolları @ ile değiştirsek?

Reply:
Projenin root folderi `src` olduğu için oradan navigate edememiştim, şimdi tekrar baktım `import VectorFaqJson from "@/../public/faq/vector.json";` olarak değiştireceğim bir tık daha temiz duruyor böyle. Root Folderi değişitirip bir yerleri patlatmayayım dedim